page_permissions.ini

```diff
@@ -0,0 +1,39 @@
+; ALLOW * ==> allow all connections (including unauthenticated)
+; ALLOW auth  ==> allows all authenticated connections
+
+[permissions]
+
+; General
+/ = ALLOW auth
+/error404 = ALLOW *
+/login = ALLOW *
+/logout = ALLOW auth
+/dashboard = ALLOW auth
+/back = ALLOW auth
+
+; Participant Search
+/agency-requests = ALLOW Superuser Administrator
+/participant-search = ALLOW Superuser Administrator
+/participant-search/* = ALLOW Superuser Administrator
+/view-participant/* = ALLOW Superuser Administrator
+
+; classes
```

Comment:
Does this mean that the User role can access these pages only Facilitators cannot?